frontend/src/stores/profileStore.jsx

```diff
@@ -1,17 +1,13 @@
 import { create } from "zustand";
 const apiEnv = import.meta.env.VITE_BACKEND_API;
+import { userStore } from "../stores/userStore";
 
 export const profileStore = create((set) => ({
   // Set the initial data
-  lastName: "",
-  firstName: "",
-  phone: "",
-  important: "",
-  color: "",
-  flower: "",
+  profile: [],
 
-  // Create a generic setter function
-  setField: (field, value) => set({ [field]: value }),
+  // Initialize the userId state
+  userID: userStore.userID,
```

Comment:
Shouldn't we be fetching the userID through:
`const userID = userStore.getState().userID` like you coded before?  And try it before switching to localStorage alternative?